### PR TITLE
fix(MangaWorld): change website URL

### PR DESCRIPTION
**Pull request by TheTecnoKing**
**Describe the changes in this pull request:**

- Updated the website's domain in `metadata.json`
- Replaced some `if-else` structures with objects and the `in` operator
- Fixed some ESLint issues

<details>
<summary> Proof showing the creation/modification is working as expected </summary>
<br>

<details>
<summary>Homepage</summary>

homepage
</details>

<details>
<summary>Bookmarks</summary>

- All
bookmarks_all
- Reading
bookmarks_reading
- Completed
bookmarks_completed
- Paused
bookmarks_paused
- Planned
bookmarks_planned
- Dropped
bookmarks_dropped
- Re-reading
bookmarks_re-reading
</details>

<details>
<summary>Archive</summary>

- Main
archive_main
- Keyword
archive_keyword
- Genre
archive_genre
- Format
archive_format
- Status
archive_status
- Author
archive_author
- Artist
archive_artist
- Year
archive_year
- Sort by
archive_sort
</details>

<details>
<summary>Reading</summary>

- Manga's page
reading_bookpage
- In the reader
reading_reader
- Other pages
reading_other
</details>

</details>

### CODE REVIEW

File: websites/M/MangaWorld/presence.ts [hunk 1]
```diff
@@ -1,10 +1,10 @@
 const presence = new Presence({
 		clientId: "860298084512038964"
 	}),
-	browsingTimestamp = Math.floor(Date.now() / 1000);
+	browsingStamp = Math.floor(Date.now() / 1000);
```
Comment (EncryptedDev) [inline websites/M/MangaWorld/presence.ts:4]: `browsingTimestamp` is preferred

File: websites/M/MangaWorld/presence.ts [hunk 2]
```diff
@@ -1,10 +1,10 @@
...
 
 presence.on("UpdateData", async () => {
-	const presenceData: PresenceData = {
+	const data: PresenceData = {
```
Comment (EncryptedDev) [inline websites/M/MangaWorld/presence.ts:7]: `presenceData` is preferred

**Review by EncryptedDev — CHANGES_REQUESTED** [review]

File: websites/M/MangaWorld/presence.ts (hunk 1 above)
Comment (TheTecnoKing) [reply websites/M/MangaWorld/presence.ts]: Fixed in commit d0770c80e834f054598a5613c64867d2763c2987

File: websites/M/MangaWorld/presence.ts (hunk 2 above)
Comment (TheTecnoKing) [reply websites/M/MangaWorld/presence.ts]: Fixed in commit d0770c80e834f054598a5613c64867d2763c2987

File: websites/M/MangaWorld/presence.ts [hunk 3]
```diff
@@ -24,23 +24,22 @@ presence.on("UpdateData", async () => {
...
-		presenceData.state = filter;
+		const categories: { [page: string]: string } = {
+				"RE-READING": "In rilettura",
+				COMPLETED: "Completati",
+				ON_HOLD: "In pausa",
+				PLAN_TO_READ: "Da leggere",
+				DROPPED: "Droppati",
+				READING: "In lettura"
+			},
+			[, , , category] = document.location.pathname.split("/");
```
Comment (EncryptedDev) [inline websites/M/MangaWorld/presence.ts:36]: Dont use array deconstruction here

**Review by EncryptedDev — CHANGES_REQUESTED** [review]

File: websites/M/MangaWorld/presence.ts (hunk 3 above)
Comment (TheTecnoKing) [reply websites/M/MangaWorld/presence.ts]: Just fixed in commit 61a617af5b30425f4ec6837912d80a1765ebe081

**Review by Slowlife01 — APPROVED** [review]

**Review by EncryptedDev — APPROVED** [review]

File: websites/M/MangaWorld/presence.ts [hunk 4]
```diff
@@ -24,23 +24,22 @@ presence.on("UpdateData", async () => {
...
-		else if (document.location.href.endsWith("DROPPED")) filter = "Droppati";
-		else if (document.location.href.endsWith("READING")) filter = "In lettura";
-
-		presenceData.state = filter;
+		const categories: { [page: string]: string } = {
+				"RE-READING": "In rilettura",
+				COMPLETED: "Completati",
+				ON_HOLD: "In pausa",
+				PLAN_TO_READ: "Da leggere",
+				DROPPED: "Droppati",
```
Comment (Fox3n) [inline websites/M/MangaWorld/presence.ts:33]: Use "Interrotti" instead of "Droppati".

File: websites/M/MangaWorld/presence.ts [hunk 5]
```diff
@@ -73,43 +71,44 @@ presence.on("UpdateData", async () => {
...
 
-		if (statusQuery === "dropped") status = "Droppati";
-		else if (statusQuery === "ongoing") status = "In corso d'opera";
-		else if (statusQuery === "completed") status = "Finiti";
-		else if (statusQuery === "paused") status = "In pausa";
-		else if (statusQuery === "canceled") status = "Cancellati";
-
-		presenceData.state = status;
+		const statuses: { [value: string]: string } = {
+				dropped: "Droppati",
```
Comment (Fox3n) [inline websites/M/MangaWorld/presence.ts:79]: Use "Interrotti" instead of "Droppati".

**Pull request merged**